src/board.cpp

```diff
@@ -19,19 +19,19 @@ Board::Board() {
 	
 	// set beginning and end of each row as buffer
 	for (int row = 0, size = BOARD::TRUE_COLUMN_SIZE; row < size; ++row) {
```

Comment:
nit: having int size = BOARD::TRUE_COLUMN_SIZE if the loop would make this easier to read.